test/powershell/Language/Scripting/Generics.Tests.ps1

```diff
@@ -0,0 +1,95 @@
+﻿Describe "Generics support" -Tags "CI" {
+    #
+    # 3 types are tested, list, stack, and dictionary.
+    #
+    # list and stack are in different assemblies, and dictionary
+    # takes more than one type parameter.
+    #
+    $listType = "system.collections.generic.list"
+    $stackType = "system.collections.generic.stack"
+    $dictionaryType = "system.collections.generic.dictionary"
+
+    It 'list[Int]' {
+        $x = [system.collections.generic.list[int]]::New()
+        $x.Add(42)
+        $x.count | Should Be 1
```

Comment:
fixed

---
In reply to: [78839631](https://github.com/Powershell/Powershell/pull/2118#discussion_r78839631) [](ancestors = 78839631)